05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+
+Some common terminology across the designs are:
+
+- sample size, generally denoted as $n$, is the number of units selected
+- population size, generally denoted as $N$, is the number of units in the population
+- sampling frame is the list of units from which the sample is drawn
+
+### Simple random sample without replacement
+
+- Description: The simple random sample (SRS) without replacement is a sampling design where a fixed sample size is selected from a sampling frame, and every possible subsample has equal probability of selection.
+- Requirements: Must have a sampling frame for entire population
+- Advantages: Simple, requires no information about the units apart from contact information
+- Disadvantages: Sampling frame may not be available for entire population. Not feasible for in-person data collection
+- Examples: Selecting lottery numbers - numbered balls are randomly sampled from a large pool and not replaced after selection
+
+#### The math {-}
+
+The estimate for the population mean of variable $y$ is:
+
+$$\bar{y}=\frac{1}{n}\sum_{i=1}^n y_i$$
+
+and the estimate of the standard error of mean is:
+
+$$se(\bar{y})=\sqrt{\frac{s^2}{n}\left( 1-\frac{n}{N} \right)}$$ and
+
+$$s^2=\frac{1}{n-1}\sum_{i=1}^n\left(y_i-\bar{y}\right)^2.$$
+
+This standard error estimate might look very similar to equations in other applications seen except for the bit on the right side of the equation $1-\frac{n}{N}$ - this is call the finite population correction factor (FPC). If the size of the frame, $N$, is very large, the FPC is negligible so it is often ignored. 
+
+To estimate proportions, we define $x_i$ as the indicator that the outcome is observed. That is, $x_i=1$ if the outcome is observed and $x_i=0$ if the outcome is not observed. Then the estimated proportion from a SRS design is:
+
+$$\hat{p}=\frac{1}{n}\sum_{i=1}^n x_i $$
+and the estimated standard error of the proportion is:
+
+$$se(\hat{p})=\sqrt{\frac{\hat{p}(1-\hat{p})}{n-1}\left(1-\frac{n}{N}\right)} $$
+
+#### The syntax {-} 
+
+If a sample that was drawn through SRS and had no nonresponse or other weighting adjustments, in R specify this design as:
+
+```{r}
+#| label: srs1
+#| eval: FALSE
+des_srs1 <- dat %>%
+  as_survey_design(fpc = fpcvar)
+```
+
+where `dat` is a tibble or data.frame with the survey data and `fpcvar` is a variable on the tibble which indicates the size of the sampling frame. If the frame was very large, sometimes the frame size is not provided. In that case, specify the design as:
```

Comment:
```suggestion
where `dat` is a tibble or data.frame with the survey data and `fpcvar` is a variable on the tibble which indicates the size of the sampling frame. If the frame was very large, sometimes the frame size is not provided. In that case, the fpc is not needed and the design can be specified as:
```

Reply:
We just discussed the fpc issue.  In the SRSWR below, you mention that it specifies the design is listed as "with replacement" because the fpc is removed.  However here, you mention that sometimes the fpc is not provided but you can still do SRS.  Add some text around how R will then consider it "with replacement" even though because it is so large it doesn't really matter.